.ci/clang.sh

```diff
@@ -38,11 +48,14 @@ build/bin/nvim --version
 make unittest
 
 # Run functional tests.
-if ! $MAKE_CMD test; then
+# FIXME (fwalch): Disabled for MSAN because of SIGPIPE error.
+if [ "$TRAVIS_OS_NAME" = "linux" -a "$CLANG_SANITIZER" -ne "MSAN" ]; then
```

Comment:
I could be spreading FUD but I believe `[ ... ] && [ ... ]` is the most reliable "and" form in [ba]sh conditionals. I also thought `-ne` was for integer-likes, not strings[1]? I would use this:

    if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$CLANG_SANITIZER" -ne "MSAN" ]; then

[1] Note that `man test` does indicate that `-ne` is for integers.